README.md

```diff
@@ -7,6 +7,22 @@ GitHub Actions [reusable workflows][reusable-workflows].
 The following section describes the usage of the different reusable workflows
 offered by this repository.
 
+### [cabal-upload](.github/workflows/cabal-upload.yml)
+
+Uploads source package and documentation to Hackage.
+
+```yml
+jobs:
+  call-cabal-upload:
+    uses: stackbuilders/reusable-workflows/.github/workflows/cabal-upload.yml@main
+    with:
+      ghc_version: "8.10"
+      cabal_version: "3.6"
```

Comment:
What do you think about adding to this example here the input `ignore_uploaded_package`? Do we want these examples to show all capabilities of a workflow, or just the most common/representative functionality?